### PR TITLE
Refactor logs download

### PR DESCRIPTION
**Pull request by yeya24**
Refactor the current log download logic.
Will download from loki every 4 minute

### CODE REVIEW

File: cmd/util/suit.go [hunk 1]
```diff
@@ -79,16 +82,20 @@ func (suit *Suit) Run(ctx context.Context) {
...
 		suit.NemesisGens,
 		suit.ClientRequestGen,
 		suit.VerifySuit,
-		loki.NewClient(fixture.Context.LokiAddress,
+		loki.NewClient(startTime, fixture.Context.LokiAddress,
 			fixture.Context.LokiUsername, fixture.Context.LokiPassword),
+		fixture.Context.LogPath,
 	)
+	if err != nil {
+		log.Fatalf("failed to create log directory %s", fixture.Context.LogPath)
```
Comment (mahjonp) [inline cmd/util/suit.go:97]: put panic into inner?

Comment (mahjonp) [reply cmd/util/suit.go:97]: or change the panic message here.

File: pkg/control/control.go [hunk 2]
```diff
@@ -488,18 +518,59 @@ func (c *Controller) queryTiDBClusterLogs(dur time.Duration, nodes []clusterType
...
+		switch n.Component {
+		case clusterTypes.TiDB, clusterTypes.TiKV, clusterTypes.PD:
+			wg.Add(1)
+
+			// containerName is added to be sure that we only fetch the logs of
+			// the container we want, not included sidecar containers.
+			go func(ns, podName, containerName string) {
+				defer wg.Done()
+				texts, err := c.lokiClient.FetchPodLogs(ns, podName, containerName,
+					"", nil, c.lastQueryTime, to, 20000, false)
```
Comment (yeya24) [inline pkg/control/control.go:553]: Is this enough? 20000 for 4 minutes. We just found one tikv produce 20w logs in 2 minutes when debug is opened @mahjonp

Comment (mahjonp) [reply pkg/control/control.go:553]: Maybe we should reduce the duration of each fetching logs? How about 20000/min?

**Comment by yeya24** [comment]
Updated @mahjonp

**Pull request merged**